common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1905,7 +1905,7 @@ def ensure_indexes(self):
         self.collection.create_index('definition.children', sparse=True)
 
         # To allow prioritizing draft vs published material
-        self.collection.create_index('_id.revision')
```

Comment:
I think you need a `background=True` in there for creating in the background.
But - are these `ensure_indexes()` methods intended to be run on production? Or locally? Should `background=True` be added to all the indices? Perhaps `ensure_indexes` should be maintained as non-production?